NPE in SummaryReporter.groovy:40

Looks like I have a build without any tasks and `longestTaskName` is `null`.

```
Caused by: java.lang.NullPointerException: Cannot invoke method plus() on null object
    at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:77)
    at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:45)
    at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
    at org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:32)
    at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
    at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
    at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
    at net.rdrei.android.buildtimetracker.reporters.SummaryReporter.formatTable(SummaryReporter.groovy:40)
    at net.rdrei.android.buildtimetracker.reporters.SummaryReporter$formatTable.callCurrent(Unknown Source)
    at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:46)
    at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
    at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:145)
    at net.rdrei.android.buildtimetracker.reporters.SummaryReporter.run(SummaryReporter.groovy:25)
    at net.rdrei.android.buildtimetracker.reporters.SummaryReporter$run.call(Unknown Source)
    at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:42)
    at net.rdrei.android.buildtimetracker.reporters.CSVReporter$run.call(Unknown Source)
    at net.rdrei.android.buildtimetracker.TimingRecorder$_buildFinished_closure1.doCall(TimingRecorder.groovy:53)
```

--- a/src/main/groovy/net/rdrei/android/buildtimetracker/reporters/SummaryReporter.groovy
+++ b/src/main/groovy/net/rdrei/android/buildtimetracker/reporters/SummaryReporter.groovy
@@ -15,6 +15,8 @@ class SummaryReporter extends AbstractBuildTimeTrackerReporter {
 
     @Override
     def run(List<Timing> timings) {
+        if (timings.size() == 0) return
+
         def threshold = getOption("threshold", "50").toInteger()
 
         if (getOption("ordered", "false").toBoolean()) {

--- a/src/test/groovy/net/rdrei/android/buildtimetracker/reporters/SummaryReporterTest.groovy
+++ b/src/test/groovy/net/rdrei/android/buildtimetracker/reporters/SummaryReporterTest.groovy
@@ -114,4 +114,14 @@ class SummaryReporterTest {
         assertTrue lines[2].contains("task3")
         assertTrue lines[3].contains("task2")
     }
+
+    @Test
+    void testEmptyTaskList() {
+        def mockLogger = new MockFor(Logger)
+        mockLogger.demand.quiet(0) {}
+
+        def reporter = new SummaryReporter([:], mockLogger.proxyInstance())
+        reporter.run([
+        ])
+    }
 }

